src/PaySimpleSdk/Helpers/Enumerations.cs

```diff
@@ -489,6 +489,52 @@ public enum PaymentScheduleType
         RecurringPayment = 2
     }
 
+    /// <summary>
+    /// CoF payment type
+    /// </summary>
+    public enum CardOnFileType
+    {
+        /// <summary>
+        /// default
+        /// </summary>
+        None = 0,
+        /// <summary>
+        /// merch initiated txn with fixed or variable amt w/o regular interval or schedule - 'credentials on file'
+        /// </summary>
+        MerchantInitiated = 1,
+        /// <summary>
+        /// cardholder initiated txn with variable amount w/o regular interval or schedule
+        /// </summary>
+        CustomerInitiated = 2,
+        /// <summary>
+        /// merch initiated txn with variable amt over regular intervals
+        /// </summary>
+        Recurring = 3,
+        /// <summary>
+        /// merch initiated txn with fixed amt over fixed period of time
+        /// </summary>
+        Installment = 4,
+    }
+
+    /// <summary>
```

Comment:
I'd take WorldPay out of this comment. Processors should be abstracted away from API users.  We want them to submit this regardless of processor, since we need to support them all (and will undoubtedly forget to update comments).

Reply:
good call, will do